### DISCUSSION TITLE
Diffusers backend fails to load model

### DISCUSSION THREAD

**Discussion by Arctomachine** [vladmandic/sdnext#2551] [Q&A]
Just after update to latest version my interface was missing checkbox for LCM sampler visibility. I tried to switch different settings and in the end this checkbox appeared, but at the same time main model stopped loading.
It is not loading at start, does nothing when clicking model in networks tab, and when trying to generate or apply LCM lora it gives this error (no more info even with debug flag)
```
Diffusers: model="Stable Diffusion" class=StableDiffusionPipeline
file="C:\sd-next\models\Stable-diffusion\revAnimated_v121.safetensors" size=1874MB

Diffusers failed loading:
model=C:\sd-next\models\Stable-diffusion\revAnimated_v121.safetensors
pipeline=Stable Diffusion/NoneType 'encoder.conv_in.weight'
```
It happens with default settings. With original backend models load normally. Diffusers pipeline value makes no difference here.

The two settings that likely were the ones I changed for LCM checkbox to appear and models to stop loading are enabling hypertile and freeu. But disabling them makes no difference, model still fails to load.

**Comment by vladmandic** [comment]
- this is issue, not discussion
- run `webui --debug` and upload actual log
